content/project-status.md

```diff
@@ -2,26 +2,47 @@
+OpenTelemetry implementations are currently in **pre-release** status. This page tracks the overall release milestones as we move towards full releases for each language SIG. You can find more details about the milestones at [this link](https://github.com/open-telemetry/opentelemetry-specification/blob/master/milestones.md).
 
-# Summary
+## Summary
 Our current goal is to provide a generally available, production quality release by the second half of 2020. Currently, we are in the _alpha_ stage. What follows is a brief explanation of what we expect to be available, when.
 
-## 0.2
-This release is expected to include a functional tracing implementation. While the API may change, you should be able to use these releases to begin exploring and understanding the API and the SDK, and the changes that have been made from OpenTracing/OpenCensus. This release also includes support for distributed context using the W3C TraceContext specification.
+### 0.2
+This release included a functional tracing implementation. While the API may change, you should be able to use these releases to begin exploring and understanding the API and the SDK, and the changes that have been made from OpenTracing/OpenCensus. This release also included support for distributed context using the W3C TraceContext specification.
 
-## 0.3
-This release is expected to contain a functional metrics implementation. Again, you may see changes to the API and the SDK after this point, but it's anticipated that these releases will allow you to explore and understand the changes that have been made from OpenCensus. In addition, this release is anticpated to include improvements to OpenTelemetry Context (such as handling baggage), various semantic conventions around trace and metric metadata, and a finalized OpenTelemetry protocol.
+### 0.3
+This release is expected to contain a functional metrics implementation. Again, you may see changes to the API and the SDK after this point, but it's anticipated that these releases will allow you to explore and understand the changes that have been made from OpenCensus. In addition, this release is anticipated to include improvements to OpenTelemetry Context, various semantic conventions around trace and metric metadata, and a finalized OpenTelemetry protocol.
 
-## 0.4
-This release is expected to contain further improvements to the tracing and metrics API and SDK in response to feedback gathered during the prior alpha releases. Additionally, we expect that the OpenTelemetry protocol will be fully implemented and languages will include an exporter to the OpenTelemetry Collector. At this phase, we anticipate all APIs will be unlikely to change.
+### Beta
```

Comment:
This potentially makes it look like `beta` is a version when 0.4 will be first beta and 0.5 will be second. Perhaps use `#### 0.4 & 0.5` and add alpha section above for 0.2 and 0.3?

Reply:
According to the [release plan](https://docs.google.com/document/d/1eviggoIguOS89dgKL-8ntUOCfPP6FLWAoQkGoBDsld0/edit?pli=1): "We’ve set the following requirements for components to reach beta: All spec version <= 0.3 features must be implemented. For APIs and SDKs this includes all 0.3 features for traces, metrics, context propagation, and resource metadata." 

It's been my impression v0.3 is the beta benchmark.  I can combine the sections: '### 0.3 & beta' as a single header if you think that will be clearer.

Reply:
I think it makes sense to combine 0.3 and beta sections as they are linked - maybe something like `0.3 - Beta`.

Reply:
sorry -- this is what I meant ^